docs/src/assets/chainrules.scss

```
@@ -1,12 +1,24 @@
 // Only for light theme
 
+@font-face {
+    font-family: JuliaMono;
+    src:
+        local('JuliaMono'),
+        url("https://cdn.jsdelivr.net/gh/cormullion/juliamono/webfonts/JuliaMono-Regular.woff2");
+}
```

Just a small style nit in the name of consistency
```suggestion
@font-face {
  font-family: JuliaMono;
  src:
    local('JuliaMono'),
    url("https://cdn.jsdelivr.net/gh/cormullion/juliamono/webfonts/JuliaMono-Regular.woff2");
}
```